Getting error java returned 1

PlayApp:
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder build
     [java] 淇℃伅: temporary project root: C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\android.keystore from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\AI2Companion-Icon.png from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\Instructions.png from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\connectwCode.png from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\connectwQR.png from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\classes.jar from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\component.json from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\components.json from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\extension.properties from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\files\AndroidRuntime.jar from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\files\component_build_info.json from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\external_comps\edu.mit.appinventor.companionextras\files\component_build_infos.json from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\mooning.png from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\assets\splash.html from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\src\edu\mit\appinventor\aicompanion3\Screen1.bky from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\src\edu\mit\appinventor\aicompanion3\Screen1.scm from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\src\edu\mit\appinventor\aicompanion3\Screen1.yail from input zip
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.ProjectBuilder extractProjectFiles
     [java] 淇℃伅: extracting C:\Users\ADMINI~1\AppData\Local\Temp\1610973860682_0.2881151188770893-0\youngandroidproject\project.properties from input zip
     [java] Component assets needed, n = 2
     [java] Component activities needed, n = 3
     [java] Component metadata needed, n = 0
     [java] Component metadata needed, n = 0
     [java] Libraries needed, n = 36
     [java] Native Libraries needed, n = 3
     [java] Permissions needed, n = 106
     [java] 涓?鏈? 18, 2021 8:44:20 涓嬪崍 com.google.appinventor.buildserver.Compiler generatePermissions
     [java] 淇℃伅: usesLocation = False
     [java] 涓?鏈? 18, 2021 8:44:22 涓嬪崍 com.google.appinventor.buildserver.Compiler writeAndroidManifest
     [java] 淇℃伅: VCode: 12926000
     [java] 涓?鏈? 18, 2021 8:44:22 涓嬪崍 com.google.appinventor.buildserver.Compiler writeAndroidManifest
     [java] 淇℃伅: VName: 2.60
     [java] java.lang.IllegalArgumentException: character to be escaped is missing
     [java]     at java.util.regex.Matcher.appendReplacement(Matcher.java:809)
     [java]     at java.util.regex.Matcher.replaceAll(Matcher.java:955)
     [java]     at java.lang.String.replaceAll(String.java:2223)
     [java]     at com.google.appinventor.buildserver.Compiler.runAaptPackage(Compiler.java:2322)
     [java]     at com.google.appinventor.buildserver.Compiler.compile(Compiler.java:1509)
     [java]     at com.google.appinventor.buildserver.ProjectBuilder.build(ProjectBuilder.java:169)
     [java]     at com.google.appinventor.buildserver.Main.main(Main.java:107)

BUILD FAILED
M:\win7\appinventor\build.xml:21: The following error occurred while executing this line:
M:\win7\appinventor\buildserver\build.xml:196: Java returned: 1

--- a/appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java
+++ b/appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java
@@ -94,6 +94,7 @@ public final class Compiler {
   private static final Object SYNC_KAWA_OR_DX = new Object();
 
   private static final String SLASH = File.separator;
+  private static final String SLASHREGEX = File.separatorChar == '\\' ? "\\\\" : "/";
   private static final String COLON = File.pathSeparator;
   private static final String ZIPSLASH = "/";
 
@@ -2319,7 +2320,7 @@ public final class Compiler {
       aaptPackageCommandLineArgs.add("--output-text-symbols");
       aaptPackageCommandLineArgs.add(symbolOutputDir.getAbsolutePath());
       aaptPackageCommandLineArgs.add("--no-version-vectors");
-      appRJava = new File(sourceOutputDir, packageName.replaceAll("\\.", SLASH) + SLASH + "R.java");
+      appRJava = new File(sourceOutputDir, packageName.replaceAll("\\.", SLASHREGEX) + SLASH + "R.java");
       appRTxt = new File(symbolOutputDir, "R.txt");
     }
     String[] aaptPackageCommandLine = aaptPackageCommandLineArgs.toArray(new String[aaptPackageCommandLineArgs.size()]);

